views/login/securityQuestion.pug

```diff
@@ -9,16 +10,15 @@ block content
   h1 #{title}
 
   form.cra-form.cra-form--lg(method='post')
-    +radios('securityQuestion', 'Answer a question from the list below', null,[
+    +radios('securityQuestion', 'Answer a question from this list to get your notice of assessment immediately after you file with this service:', securityQuestion,[
       {value: "/login/questions/child", label: "Your child’s name and birthdate"},
       {value: "/login/questions/addresses", label: "Your last two mailing addresses before your current address"},
-      {value: "/login/questions/prison", label: "Your date of entry or release from prison"},
-      {value: "/login/questions/dateOfResidence", label: "Date you became a resident of Canada"},
-      {value: "/login/questions/bankruptcy", label: "Details of your bankruptcy assignment"}
+      {value: "/login/questions/bank", label: "Your bank or credit union account information"},
+      {value: "/login/questions/taxReturn", label: "Your total income reported on a previous tax return"},
+      {value: "/checkAnswers", label: "Skip security questions"}
     ])
 
-    div
-      p
-        a(href="/login/securityQuestion2") #{__('I can’t answer these questions')}
+    +banner('blue')
+      p #{__('If you skip these questions, you will wait about 8 weeks to get a notice of assessment.')}
```

Comment:
Has this been updated to "2 weeks"?

Reply:
talked to Brian and we're going to leave as is for now